.github/workflows/go.yml

```diff
@@ -0,0 +1,31 @@
+name: Go
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+
+      - name: Install dependencies
+        run: go get .
+      - name: Test with the Go CLI
+        run: go test ./...
+
+  golangci:
+      name: lint
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v4
+        - uses: actions/setup-go@v5
+          with:
+            go-version-file: 'go.mod'
+        - name: golangci-lint
+          uses: golangci/golangci-lint-action@v6
+          with:
+            version: v1.59
```

Comment:
Is that possible to the the version defined in go.mod?

Reply:
yes I changed